data/lists.js

```diff
@@ -3,3 +3,168 @@
 // edit list - used by /list/:id route
 // delete list - used by /list/:id route
 // delete entry from list - used by /list/:id route
+import { users } from "../config/mongoCollections.js";
+import { ObjectId } from "mongodb";
+import * as validation from "../utils/validation.js";
+
+const exportedMethods = {
+  async create(userId, title, movies) {
+    //for validation validate if
+    //userId is valid
+    //title is a string
+    //movies is an array of integers
+    validation.checks(userId);
+    userId = userId.trim();
+    if (!ObjectId.isValid(userId)) throw "invalid object ID";
+    title = title.trim();
```

Comment:
There is no validation here for the title, you are just trimming it. You could use the validation helper to check if its a string and trim it:
```suggestion
    title = validation.checkString(title, "Title");
```